pandas/core/indexes/multi.py

```diff
@@ -68,6 +68,23 @@ class MultiIndex(Index):
         Copy the meta-data
     verify_integrity : boolean, default True
         Check that the levels/labels are consistent and valid
+
+    Examples
+    ---------
+    A new ``MultiIndex`` is typically constructed using one of the helper
+    methods :attr:`MultiIndex.from_arrays``, :attr:`MultiIndex.from_tuples``
+    and :attr:`MultiIndex.from_product``. For example (using ``.from_arrays``):
+
+    >>> arrays = [[1, 1, 2, 2], ['red', 'blue', 'red', 'blue']]
+    >>> mi = MultiIndex.from_arrays(arrays, names=('number', 'color'))
+    >>> mi
+
+    Such a ``MultiIndex`` instance can be used as an index in a Series or
```

Comment:
this last part is not really relevant

you can certainly link to a MI (and Index) from Series / DataFrame constructor though